install/docker-extension/docker-compose.yaml

```diff
@@ -1,5 +1,10 @@
 version: '3'
 services:
+  meshery-proxy:
+    image: ${DESKTOP_PLUGIN_IMAGE}
+    cap_add:
+      - DAC_OVERRIDE
```

Comment:
See the description of `dac_override` on this page: https://www.redhat.com/en/blog/secure-your-containers-one-weird-trick. They are not kind to the need for this capability.

Reply:
🤞 for no blow back.